server/modules/health-checks/index.js

```diff
@@ -5,7 +5,8 @@
 let resultCodes = require('./resultCodes');
```

Comment:
I had a quick check that the file name was resultCodes and not resultcodes... while that looked ok, perhaps it should be a pre commit on file names for the team to prevent this popping up in the future.